### PR TITLE
Create jest specs, close #12

### LINKED ISSUE #12: create basic jest specs to test rendering
Now that the basic POC is complete, it would be nice to have a test suite to test future changes.

A great way to test that your React components render, and also test that they render the same way across changes, is snapshot tests. There are links to all the tutorials and guides you should need to set up Jest and write basic snapshot tests at the bottom of this issue.

We need a single snapshot test for each component. Each test will end up looking something like:
```js
it('renders and nothing has changed', () => {
  const dots = renderer.create(<Dots />).toJSON();
  expect(dots).toMatchSnapshot();
});
```
[Here is a link to the official Jest setup instructions](https://jestjs.io/docs/en/getting-started)
[Here is a link with a more in depth description about snapshot tests](https://jestjs.io/docs/en/snapshot-testing)
[Here is a link with a good amount of official examples](https://github.com/facebook/jest/blob/master/examples/snapshot/__tests__/link.react.test.js)

### PR DESCRIPTION
**Pull request by roskill**

### CODE REVIEW

File: src/containers/__tests__/index.test.js.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { Atebits } from '../atebits.js';
```
Comment (ajwann) [inline src/containers/__tests__/index.test.js.jsx]: Hey all of these look like perfect snapshot tests, great work.

File: .eslintrc.js [hunk 2]
```diff
@@ -2,11 +2,9 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
```
Comment (ajwann) [inline .eslintrc.js]: Out of curiosity, what command did you run that replaced all the double quotes here with single quotes?

File: .eslintrc.js [hunk 3]
```diff
@@ -2,11 +2,9 @@ module.exports = {
   env: {
```
Comment (ajwann) [inline .eslintrc.js]: Out of curiosity, what command did you run that replaced all the double quotes here with single quotes?

File: .eslintrc.js [hunk 4]
```diff
@@ -2,11 +2,9 @@ module.exports = {
...
     browser: true,
     es6: true,
+    jest: true,
```
Comment (ajwann) [inline .eslintrc.js]: Out of curiosity, what command did you run that replaced all the double quotes here with single quotes?

File: .eslintrc.js [hunk 5]
```diff
@@ -18,16 +16,14 @@ module.exports = {
...
     sourceType: 'module',
   },
-  plugins: [
-    'react',
-  ],
+  plugins: ['react'],
   rules: {
-    "import/no-extraneous-dependencies": "off" ,
-    "import/prefer-default-export": "off"
+    'import/no-extraneous-dependencies': 'off',
```
Comment (ajwann) [inline .eslintrc.js]: Out of curiosity, what command did you run that replaced all the double quotes here with single quotes?

Comment (roskill) [reply .eslintrc.js]: I use Prettier as a VS Code extension with the following config:

```
{
  "tabWidth": 2,
  "useTabs": false,
  "singleQuote": true,
  "trailingComma": "all",
  "printWidth": 100
}
```

Comment (ajwann) [reply .eslintrc.js]: Ahh ok, that explains it!

File: src/containers/__tests__/index.test.js.jsx (hunk 1 above)
Comment (roskill) [reply src/containers/__tests__/index.test.js.jsx]: Thanks! Happy to have made my first contribution to someone's project 😃

**Pull request merged**